README.rst

```diff
@@ -6,21 +6,17 @@
    :target: https://docs.ansible.com/ansible-lint/rules/default_rules.html
    :alt: Ansible-lint rules explanation
 
-.. image:: https://img.shields.io/badge/Code%20of%20Conduct-Ansible-silver.svg
+.. image:: https://img.shields.io/badge/Code%20of%20Conduct-silver.svg
    :target: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
    :alt: Ansible Code of Conduct
 
 .. image:: https://img.shields.io/badge/Mailing%20lists-Ansible-orange.svg
    :target: https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information
    :alt: Ansible mailing lists
 
-.. image:: https://github.com/ansible/ansible-lint/workflows/%F0%9F%91%B7/badge.svg
-   :target: https://github.com/ansible/ansible-lint/actions?query=workflow%3A%F0%9F%91%B7
-   :alt: 👷 GitHub Actions CI/CD build status — tests
-
-.. image:: https://github.com/ansible/ansible-lint/workflows/%F0%9F%9A%A8/badge.svg
-   :target: https://github.com/ansible/ansible-lint/actions?query=workflow%3A%F0%9F%9A%A8
-   :alt: 🚨 GitHub Actions CI/CD build status — linters
+.. image:: https://github.com/ansible/ansible-lint/workflows/gh/badge.svg
+   :target: https://github.com/ansible/ansible-lint/actions?query=workflow%3Agh
```

Comment:
How about adding a branch filter (`&branch=master`) and maybe event (`push`)? I think this would filter out some builds that may be confusing to folks who are unfamiliar with the UI.

Reply:
Really good point, both. It is likely that a visitor may want to see CI results from push actions made to master branch only.